### PR TITLE
MM-20996 Bump mattermost-redux version.

### PR DESCRIPTION
**Pull request by crspeller**
#### Summary
Bumping redux version to fix archive channel permalinks causing blank screen. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-20996

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2305,9 +2305,9 @@
       }
     },
     "@react-native-community/netinfo": {
-      "version": "4.6.1",
-      "resolved": "https://registry.npmjs.org/@react-native-community/netinfo/-/netinfo-4.6.1.tgz",
-      "integrity": "sha512-RIcrNzVnkes6/d5jFprce8i6ruUO9+/FVZZgejlu/TSnysyFXHNJKgcIqxbKx+7XfQxfjrCGeCvsfYxrFrU0zw=="
+      "version": "4.7.0",
```
Comment (jespino) [inline package-lock.json]: This change is accidental or on purpose?

Comment (crspeller) [reply package-lock.json]: I just ran `npm install`.

**Comment by enahum** [comment]
what about https://github.com/mattermost/mattermost-webapp/pull/4513 ?

**Comment by crspeller** [comment]
@enahum I didn't see it. Will close and wait for that one.

**Pull request closed without merging**